source/speech/__init__.py

```diff
@@ -1003,9 +1058,11 @@ def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=cont
 			# Indentation must be spoken in the default language,
 			# but the initial format field specified a different language.
 			# Insert the indentation before the LangChangeCommand.
-			speechSequence.insert(-1, indentationSpeech)
+			langChange = speechSequence.pop()

```

Comment:
I actually realized that this can be done in one line instead of three.

```suggestion
			speechSequence[-1:-1] = = indentationSpeech)
```

Example:

```
>>> l=[1, 2, 3]
>>> l[-1:-1] = [4,5,6]
>>> l
[1, 2, 4, 5, 6, 3]
```